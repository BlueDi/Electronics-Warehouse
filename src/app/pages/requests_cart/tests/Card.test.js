import React from 'react';
import CardItem from '../Card';
import renderer from 'react-test-renderer';

var item = {
  description: 'BMW motor',
  details: { details: 'Super detailed description', manufacturer: 'Multicomp' },
  free_stock: 1,
  image:
    'iVBORw0KGgoAAAANSUhEUgAAAG8AAABYCAYAAAD7uql1AAAgAElEQVR4Xu29e5RV5Znu+5vfN6/r\nVmtVFXWjoChABFERVCKgUUAjthFzMZqYxLjT3aP79D456d49zj5nZJzTo0f/0+fsHumT7O6xY/be\n2umklcSEaFRuIiiIF0AUFQVFQSAlRRVVRdW6zut3/phrLsâ€¦',
  last_edit: '2018-11-20T11:46:01.628Z',
  last_price: 89.12,
  location: 'HHD12U',
  reference: '213748227'
};

test('Create a Card the Request Cart List', () => {
  const component = renderer.create(<CardItem item={item} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
