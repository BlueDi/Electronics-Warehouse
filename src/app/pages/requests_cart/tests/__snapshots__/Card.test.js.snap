// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create a Card the Request Cart List 1`] = `
<div
  className="ui card"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    />
    <div
      className="meta"
    >
      Location: 
      HHD12U
    </div>
    <div
      className="description"
    >
      <p>
        description
        : 
        BMW motor
      </p>
      <p>
        free_stock
        : 
        1
      </p>
      <p>
        last_edit
        : 
        2018-11-20T11:46:01.628Z
      </p>
      <p>
        last_price
        : 
        89.12
      </p>
      <p>
        reference
        : 
        213748227
      </p>
      <div
        className="ui divider"
      />
      <div
        className="ui divider"
      />
      <p>
        details
        : 
        Super detailed description
      </p>
      <p>
        manufacturer
        : 
        Multicomp
      </p>
    </div>
  </div>
  <div
    className="extra content"
  >
    Available: 
    1
  </div>
</div>
`;
